G-Earth/src/main/resources/gearth/ui/translations/messages_fr.properties

```diff
@@ -11,65 +11,65 @@ tab.connection.client=Type de client:
-tab.connection.state.notconnected=Pas connecté
+tab.connection.version=Version de l'Hotel:
+tab.connection.state=Status:
+tab.connection.state.notconnected=Non connecté
 tab.connection.state.connected=Connecté
 tab.connection.state.waiting=En attente de connexion
 tab.connection.port=Port:
-tab.connection.host=Host:
+tab.connection.host=Nom d'hôte:
 tab.connection.button.connect=Connecter
 tab.connection.button.abort=Annuler
-tab.connection.autodetect=Détection auto
+tab.connection.autodetect=Détection Auto
 
 ### Tab - Injection
 tab.injection=Injecter
 tab.injection.corrupted=isCorrupted
 tab.injection.corrupted.true=True
 tab.injection.corrupted.false=False
-tab.injection.description.header=header
-tab.injection.description.packets=packets
+tab.injection.description.header=En-tête
+tab.injection.description.packets=Paquets
 tab.injection.description.id=id
 tab.injection.description.length=taille
 tab.injection.send.toserver=Envoyer au serveur
 tab.injection.send.toclient=Envoyer au client
 tab.injection.history=Historique:
 tab.injection.history.clear=Effacer
-tab.injection.history.tooltip=Double clique un packet pour le restaurer
-tab.injection.log.packetwithid=Packet avec id
+tab.injection.history.tooltip=Double clique sur un paquet pour le restaurer
+tab.injection.log.packetwithid=Paquet avec un id
 
 ### Tab - Tools
 tab.tools=Outils
-tab.tools.button.encode=Encode
-tab.tools.button.decode=Decode
+tab.tools.button.encode=Chiffrer
+tab.tools.button.decode=Déchiffrer
 tab.tools.type.integer=Integer
 tab.tools.type.ushort=Ushort
-tab.tools.encoding=Encoder
-tab.tools.decoding=Decoder
-tab.tools.packet=Packet
+tab.tools.encoding=Chiffrer
+tab.tools.decoding=Déchiffrer
+tab.tools.packet=Paquet
 tab.tools.expression=Expression
 ### Tab - Scheduler
 tab.scheduler=Planificateur
 tab.scheduler.table.index=Index
-tab.scheduler.table.packet=Packet
+tab.scheduler.table.packet=Paquet
 tab.scheduler.table.interval=Interval
 tab.scheduler.table.destination=Destination
 tab.scheduler.table.edit=Modifier
-tab.scheduler.setup.packet=Packet
-tab.scheduler.setup.interval=Interval
+tab.scheduler.setup.packet=Paquet
+tab.scheduler.setup.interval=Intervalle
 tab.scheduler.button.clear=Effacer
-tab.scheduler.button.clear.tooltip=Effacer tous
-tab.scheduler.button.save=S
+tab.scheduler.button.clear.tooltip=Tout effacer
+tab.scheduler.button.save=Sauvegarder
 tab.scheduler.button.save.tooltip=Sauvegarder dans un fichier
 tab.scheduler.button.save.windowtitle=Sauvegarder le planificateur dans un fichier
-tab.scheduler.button.load=L
+tab.scheduler.button.load=Charger
 tab.scheduler.button.load.tooltip=Charger depuis un fichier
 tab.scheduler.button.load.windowtitle=Charger le planificateur depuis un fichier
 tab.scheduler.button.add=Ajouter
 tab.scheduler.button.edit=Modifier
-tab.scheduler.direction.in=In
-tab.scheduler.direction.out=Out
-tab.scheduler.hotkeys=Activer raccourcis (Ctrl+Shift+Index)
+tab.scheduler.direction.in=Entrant
```

Comment:
keep it short like "in" and "out" (keep it "in" and "out if that doesn't work in french